google-cloud-storage_batch_operations/samples/storage_batch_cancel_job.rb

```diff
@@ -0,0 +1,39 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START storage_batch_cancel_job]
+require "google/cloud/storage_batch_operations"
+
+def cancel_job project_name:, job_name:
```

Comment:
I think we should stick to `project_id` instead of `project_name`. `project_name` can be confusing.